packages/dina-ui/intl/seqdb-fr.ts

```diff
@@ -88,6 +89,7 @@ export const SEQDB_MESSAGES_FRENCH: Partial<typeof SEQDB_MESSAGES_ENGLISH> = {
   selectAllCheckedSamplesButtonText:
     "Sélectionner tous les échantillons cochés -->",
   selectButtonText: "Sélectionner",
+  selectMaterialSamples: "Sélectionner échantillons matériaux",
```

Comment:
English translations are given to a service that will do the translations for us, these translations should be removed so they can be done by the service.